app/src/components/annotation/ContinuousAnnotationInput.tsx

```diff
@@ -19,49 +19,37 @@ type ContinuousAnnotationInputProps =
 export const ContinuousAnnotationInput = forwardRef<
   HTMLDivElement,
   ContinuousAnnotationInputProps
->(
-  (
-    {
-      annotationConfig,
-      containerRef,
-      annotation,
-      onSubmitExplanation,
-      ...props
-    },
-    ref
-  ) => {
-    return (
-      <Flex gap="size-50" alignItems="center" position="relative">
-        <AnnotationInputExplanation
-          annotation={annotation}
-          onSubmit={onSubmitExplanation}
-          containerRef={containerRef}
+>(({ annotationConfig, annotation, onSubmitExplanation, ...props }, ref) => {
+  return (
+    <Flex gap="size-50" alignItems="center" position="relative">
+      <AnnotationInputExplanation
+        annotation={annotation}
+        onSubmit={onSubmitExplanation}
+      />
+      <NumberField
+        defaultValue={annotation?.score ?? undefined}
+        {...props}
+        ref={ref}
```

Comment:
Was a little confused why the ref is placed here - makes sense but just noting my confusion.